ios/MusicControlManager.m

```diff
@@ -9,21 +9,56 @@
 @interface MusicControlManager ()
 
 @property (nonatomic, copy) NSString *artworkUrl;
+@property (nonatomic, strong) NSDictionary *mediaKeys;
 
 @end
 
+NSString * const MEDIA_PLAYBACK_RATE = @"playbackRate";
+
 @implementation MusicControlManager
 
 @synthesize bridge = _bridge;
 
 RCT_EXPORT_MODULE()
 
+- (NSDictionary *) mediaKeys {
```

Comment:
I tried in XCode and no error was raised. That's why I'm asking that. At least can you make the method at the classe level instead of the instance level.